tests/functional/test_basic_behavior.py

```diff
@@ -0,0 +1,96 @@
+"""Basic behavior feature tests."""
+import pytest
+from pytest_bdd import (
+    given,
+    scenario,
+    then,
+    when,
+)
+
+import os
+import subprocess 
+from urllib.parse import urljoin
+import time
+
+@pytest.fixture(scope='session')
+def splinter_headless():
+    """Override splinter headless option."""
+    if os.environ['HEADLESS'] == "true":
+        return True
+    else:
+        return False 
+
+@pytest.fixture(scope='session')
+def splinter_webdriver():
+    """Override splinter webdriver name."""
+    return 'chrome'
+
+# Fixture to save information to use through steps
+@pytest.fixture
+def step_context():
+    return {}
```

Comment:
```suggestion
    """Fixture to save information to use through steps."""
    return {}
```

Reply:
Agree, this is more pythonish instead of my ruby background.